REFERENCE.md

```diff
@@ -7918,6 +7952,14 @@ Data type: `Any`
 
 Default value: `$apache::params::ssl_certs_dir`
 
+##### <a name="ssl_reload_on_change"></a>`ssl_reload_on_change`
```

Comment:
Somehow this type has no docs, but I don't see why. The code has a parameter.

Reply:
@ekohl I don't get what you mean, i see nothing wrong...?

Reply:
Normally I would expect a description. However, now I don't see any text.

Reply:
I still don't get it. Other parameters don't have any text as well in that section. What exactly are you missing?

Reply:
@ekohl Can you please give feedback here?

Reply:
I traced it down to https://github.com/puppetlabs/puppetlabs-apache/pull/2165.

Reply:
ekohl: Good finding. How should we proceed? Should i merge your commit into my branch here and after that regenerate REFERENCE.md (resulting in 3 commits)? 
Or is it okay, when i do the simple fix by myself and regenerate REFERENCE.md as well and put it into one commit (like i usually did here by force pushes)?

Reply:
It'd be great if you could rebase on my PR. I now regenerated REFERENCE.md in it so your diff should remain small.